python/google/protobuf/json_format.py

```diff
@@ -66,7 +66,7 @@ class ParseError(Exception):
   """Thrown in case of parsing error."""
 
 
-def MessageToJson(message, including_default_value_fields=False):
+def MessageToJson(message, including_default_value_fields=False, indent=None):
```

Comment:
Please keep the original default value "2". Otherwise you will need to update the tests and that will be too much unnecessary changes.